.github/workflows/ci.yml

```
@@ -17,18 +17,28 @@ jobs:
         with:
           version: 1
   build:
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-latest
```

```suggestion
    runs-on: ubuntu-18.04
```

Trying to switch back to fixed version due to failing integration tests